src/Core/cldj_init_mod.F90

```diff
@@ -44,11 +46,12 @@ subroutine INIT_CLDJ (AMIROOT,DATADIR,NLEVELS,TITLEJXX,NJXU,NJXX)
 
       if (AMIROOT) write(6,*) ' Solar/Cloud-J  ver-7.7 initialization'
 
-#if defined ( MODEL_GEOSCHEM )
+#ifndef CLOUDJ_STANDALONE )
```

Comment:
Remove the `)`, it's not needed for `#ifndef`

Reply:
Fixed.